tools/build_defs/scala/scala.bzl

```diff
@@ -95,6 +108,11 @@ def _collect_jars(ctx):
       jars += target.java.transitive_runtime_deps
   return jars
 
+def _args_for_suites(suites):
+  args = ["-o"]
+  for suite in suites:
+    args.extend(["-s", suite])
```

Comment:
I agree that it would be nice. As discussed on the email thread, the big challenge is that, in Scala, the test suite name cannot easily be inferred from the path (because packages are independent of the file sytem). So I see two options:
- Add a custom test runner (written in Scala/Java) which wraps the usual scalatest runner and does this discovery at runtime.
- Specify the suites by hand.

I went for the second option for now because the first one seemed controversial on the thread. Also, supporting the second one seems like a good idea regardless of whether we end up adding the first one in addition.

What do you think?